services/response-fixtures.js

```diff
@@ -1,9 +1,9 @@
 'use strict'
 
-const invalidJSON = function() {
```

Comment:
Do you think it would be better to keep `function invalidJSON() {` here rather than in `module.exports`?

Reply:
They seem equally clear to me. I moved it inline since it's more of a fixture than a real function.

Reply:
No worries :+1: